setup.cfg

```diff
@@ -79,6 +83,8 @@ filterwarnings =
   ignore:Deprecation Warning.*timeout.*parameter no longer needed:UserWarning
   # This is from skimage import
   ignore:.*np\.bool8.*is a deprecated alias:DeprecationWarning
+  #often appears during MRT calculation but is inconsequential
+  ignore:All-NaN slice encountered:RuntimeWarning
```

Comment:
I feel like if you don't care about such warning, it should be handled properly during the calculation in the code, not as blanket ignore for test suite that also affect other modules. Please have a look to see if `np.seterr` is what you really want and use it as context manager to the affected calculation only.

Reply:
I now use warnings.filterwarnings for that specific warning, only inside the trailfinder class. It's very common due to how missing data are presented in the module, but it is inconsequential.